### ISSUE TITLE
Suggestion: rephrase funder selection when creating a DMP template

### ISSUE THREAD

**Issue by maarten-vermeyen** [DMPbelgium/roadmap#59]
Hi Everyone,

I have a small suggestion for improving the clarity for end users, but I'm not certain that other organizations have the same problems or whether they handle this differently than UAntwerp. 

Currently, the BOF and IOF funding channels of the different Universities are not configured as funders. I believe this is done so that different institutions that have BOF/IOF funding can implement their own organizational templates. However, this is confusing to end users, as they have to select "no funder" in order to get the correct organizational templates.

If others agree, I would like to suggest to change the strings on the page https://dmponline.be/plans/new:

* **_Select the primary funding organisation_** =>  **_Select the primary external funding organisation_**
* **_No funder associated with this plan or my funder is not listed_** => **_No external funder associated with this plan or my funder is not listed_**

These would have to be set appropriately in the relevant locale configs.

As stated, this issue is mostly relevant to institutions with internal funding bodies (i.e. only the universities?), so this would definitely have to be discussed with the wider user group.

Kind regards,

Maarten

**Comment by nicolasfranck** [comment]
I read another confusion here too: the selection of the funder and/or organisation during the plan creation wizard has nothing to do with your real funder. It is all about the selection of the right template. The wizard filters the existing templates based on the selected funder and/or organisation; yes that is not clear from the interface, I agree. That is going to get better in the future. 

So if you do not find your funder, don't worry; just find one that has the right questions for you.

It is a questionnaire by the way, not a duplicate of your internal administration ;-)

**Comment by maarten-vermeyen** [comment]
I think the underlying cause for the confusion for our users is that that an organisation can be be both a funder and a research performing organisation. Our users in this case would expect the list of funders to contain either _'University of Antwerp'_ or the separate funding bodies _[ 'University of Antwerp - Special Research Fund',  'Ghent University - Special Research Fund', ...]_.  How did you set this up at UGhent?

If there are already changes planned on the template selection wizard that make things more clear, that would be great. I would prefer a single selection list based on all the template that are applicable for an organisation instead of first having to select the funder.

At the moment, we've tried adding another template to our organisational templates. We used to have only a single published template. In this case the interface didn't show the template name when the "no funder or not listed" box was checked indicating that the funder wasn't listed. By adding a second template to our organization, a selection box appears with the template names, so it is at least clear for users who get that far that they will be presented with the correct template for BOF funded research.
